### PR TITLE
Feature/ob4

### PR DESCRIPTION
**Pull request by varex83**

### CODE REVIEW

File: data/generators/extensions.js [hunk 1]
```diff
@@ -204,10 +211,15 @@ export function generateExtension(extensionName, standardName, contractName, ver
       accessControlEnumerableExtension.setStorage(accessControlEnumerableStorage.getStorage())
 
       accessControlEnumerableExtension.setImpl(new TraitImpl(`AccessControlEnumerable`, contractName, additionalMethods))
-      accessControlEnumerableExtension.addConstructorAction('_instance._init_with_admin(_instance.env().caller());')
-      accessControlEnumerableExtension.addConstructorAction(
+      // accessControlEnumerableExtension.addConstructorAction('_instance._init_with_admin(_instance.env().caller());')
```
Comment (o-tsaruk) [inline data/generators/extensions.js:214]: do we need this comment?

**Review by o-tsaruk — DISMISSED** [review]

File: components/WizardOutput.js [hunk 2]
```diff
@@ -172,10 +172,18 @@ const versionInfo = {
     scaleInfoVersion: '2.3',
     brushDeclaration: (features) =>
         `openbrush = { tag = "3.0.0", git = "https://github.com/727-Ventures/openbrush-contracts", default-features = false, features = [${features}] }`
+  },
+  'v4.0.0': {
+    edition: '2021',
+    inkVersion: '4.1.0',
```
Comment (coreggon11) [inline components/WizardOutput.js:178]: should be 4.2.1

File: components/WizardOutput.js [hunk 3]
```diff
@@ -172,10 +172,18 @@ const versionInfo = {
     scaleInfoVersion: '2.3',
     brushDeclaration: (features) =>
         `openbrush = { tag = "3.0.0", git = "https://github.com/727-Ventures/openbrush-contracts", default-features = false, features = [${features}] }`
+  },
+  'v4.0.0': {
```
Comment (coreggon11) [inline components/WizardOutput.js:176]: should be 4.0.0-beta for now

File: components/Wizard.js [hunk 4]
```diff
@@ -109,6 +109,7 @@ const Wizard = () => {
                       }}
                       defaultValue={'v3.0.0'}
                     >
+                      <option value="v4.0.0">v4.0.0</option>
```
Comment (coreggon11) [inline components/Wizard.js:112]: should be 4.0.0-beta for now

**Review by coreggon11 — CHANGES_REQUESTED** [review]
gj, left small comments

**Review by coreggon11 — APPROVED** [review]

**Pull request merged**